fsharp-backend/src/LibBinarySerialization/SerializedTypes.fs

```diff
@@ -91,17 +91,17 @@ module FQFnName =
     | Package of PackageFnName
 
 [<MessagePack.MessagePackObject>]
-type Pattern =
-  | PVariable of id * string
-  | PConstructor of id * string * List<Pattern>
-  | PInteger of id * int64
-  | PBool of id * bool
-  | PCharacter of id * string
-  | PString of id * string
-  | PFloat of id * Sign * string * string
-  | PNull of id
-  | PBlank of id
-  | PTuple of id * Pattern * Pattern * List<Pattern>
+type MatchPattern =
```

Comment:
Is renaming the type allowed?

I guess we must have tests, so it must be?

Reply:
Yes, it seems to be. The order is what matters afaict